docs/gettingstarted.rst

```diff
@@ -148,23 +195,26 @@ Deploy on CERN infrastructure
       $ virtualenv reana
       $ source reana/bin/activate
 
-7. Install `reana-cluster`:
+8. Install `reana-cluster`:
 
    .. code-block:: console
 
       (reana) $ pip install reana-cluster
 
-8.  Set the database URI and instantiate REANA cluster:
+9. Instantiate REANA cluster:
+
+   Edit ``reana-cluster.yaml`` adding the ``cephfs_monitors`` obtained
+   in the step 5 and instatiate the cluster.
 
    .. code-block:: console
 
-      (reana) $ reana-cluster init
+      (reana) $ reana-cluster -f reana-cluster.yaml --prod init
```

Comment:
I wonder whether `--prod` is an appropriate name, instead of something more functional.... Because one man's developer server could be another man's production :wink: But that would be a bigger change, not worth doing now... Just jotting down thoughts upon reading the diff.